<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0052)http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="StdNCol">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="diagpath"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3073211/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology</title>
        
        <!-- AppResources external_resources begin -->
        <script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/jig.min.js"></script><link rel="stylesheet" type="text/css" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/jig.core.theme.min.css"><script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/jquery.ui.widgets.min.js"></script><link type="text/css" rel="stylesheet" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/jquery.ui.widgets.min.css">

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology.htm"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Diagnostic Pathology"><meta name="citation_title" content="Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology"><meta name="citation_authors" content="Tomasz Markiewicz"><meta name="citation_date" content="2011"><meta name="citation_issue" content="Suppl 1"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="S18"><meta name="citation_doi" content="10.1186/1746-1596-6-S1-S18"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3073211/?report=abstract"><meta name="citation_pmid" content="21489188"><meta name="DC.Title" content="Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="Tomasz Markiewicz"><meta name="DC.Date" content="2011"><meta name="DC.Identifier" content="10.1186/1746-1596-6-S1-S18"><meta name="DC.Language" content="en"><link rel="stylesheet" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/common.min.js"></script><script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/NcbiTagServer.min.js"></script><script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-diagpath.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><link rel="alternate" type="application/epub+zip" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/epub/"><link rel="alternate" type="application/pdf" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/pdf/1746-1596-6-S1-S18.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="http://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="8A1A71E23F554321000000000096E055"><link type="text/css" rel="stylesheet" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/3855312.css"><link type="text/css" rel="stylesheet" href="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/3929378.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC3073211%2F&ncbi_pdid=article&ncbi_phid=8A1A71E23F554321000000000096E055'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style></head>
    <body class="article">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="http://www.ncbi.nlm.nih.gov/corehtml/query/static/unsupported-browser.html#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="http://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="http://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="http://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="http://www.ncbi.nlm.nih.gov/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="http://www.ncbi.nlm.nih.gov/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3073211%2F" id="sign_in">Sign in to NCBI</a><a href="http://www.ncbi.nlm.nih.gov/account/signout/?back_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3073211%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="http://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <form method="get" action="http://www.ncbi.nlm.nih.gov/pmc/" id="entrez-search-form">
  <input type="hidden" id="ncbisearchbar-app-name" value="pmc">
  <div class="search">
    <label for="term" class="offscreen_noflow">Search term</label>
    <label for="entrez-search-db" class="offscreen_noflow">Search database</label>
    <div class="search_form">
      <select id="entrez-search-db" name="resource">
        <option value="gquery">All Databases</option>
        <option value="pubmed">PubMed</option>
        <option value="protein">Protein</option>
        <option value="nuccore">Nucleotide</option>
        <option value="nucgss">GSS</option>
        <option value="nucest">EST</option>
        <option value="structure">Structure</option>
        <option value="genome">Genome</option>
        <option value="assembly">Assembly</option>
        <option value="bioproject">BioProject</option>
        <option value="biosample">BioSample</option>
        <option value="books">Books</option>
        <option value="cdd">Conserved Domains</option>
        <option value="clinvar">ClinVar</option>
        <option value="clone">Clone</option>
        <option value="gap">dbGaP</option>
        <option value="dbvar">dbVar</option>
        <option value="epigenomics">Epigenomics</option>
        <option value="gene">Gene</option>
        <option value="gds">GEO DataSets</option>
        <option value="geoprofiles">GEO Profiles</option>
        <option value="homologene">HomoloGene</option>
        <option value="medgen">MedGen</option>
        <option value="mesh">MeSH</option>
        <option value="ncbisearch">NCBI Web Site</option>
        <option value="nlmcatalog">NLM Catalog</option>
        <option value="omim">OMIM</option>
        <option value="pmc" selected="selected" data-ac-dict="pmc-search-autocomplete">PMC</option>
        <option value="popset">PopSet</option>
        <option value="probe">Probe</option>
        <option value="proteinclusters">Protein Clusters</option>
        <option value="pcassay">PubChem BioAssay</option>
        <option value="biosystems">BioSystems</option>
        <option value="pccompound">PubChem Compound</option>
        <option value="pcsubstance">PubChem Substance</option>
        <option value="pubmedhealth">PubMed Health</option>
        <option value="snp">SNP</option>
        <option value="sra">SRA</option>
        <option value="taxonomy">Taxonomy</option>
        <option value="toolkit">ToolKit</option>
        <option value="toolkitall">ToolKitAll</option>
        <option value="toolkitbook">ToolKitBook</option>
        <option value="unigene">UniGene</option>
      </select>
      <div class="nowrap">
        <div class="nowrap">
          <div class="jig-ncbiclearbutton-wrap"><input name="term" type="text" id="term" value="" class="jig-ncbiautofocus jig-ncbiclearbutton jig-ncbiautocomplete" autocomplete="on" config="dictionary: &#39;pmc-search-autocomplete&#39;,disableUrl:&#39;?session$autocomplete=false&#39;" title="Use up and down arrows to choose an item from the autocomplete." aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" style="visibility: hidden;"><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/clear.png" alt="Clear input"></a></div>
        </div>
        <button type="submit" id="search" class="button_search nowrap">Search</button>
      </div>
    </div>
    <ul class="searchlinks inline_list">
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/limits/">Limits</a>
                    </li>
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li>
      <li>
                        <a href="http://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li>
      <li class="help">
                        <a target="_blank" href="http://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li>
    </ul>
  </div>
</form>

</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="http://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="archive-page"><a class="navlink" href="http://www.ncbi.nlm.nih.gov/pmc/journals/414/">Diagn Pathol</a></li><li class="issue-page"><a class="navlink" href="http://www.ncbi.nlm.nih.gov/pmc/issues/195730/">v.6(Suppl 1); 2011</a></li><li class="accid">PMC3073211</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/logo-diagpath.png" alt="Logo of diagpath" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,162,75" alt="BioMed Central" title="BioMed Central" href="http://www.biomedcentral.com/" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,99,74" alt="Biomed Central Web Site" title="Biomed Central Web Site" href="http://www.biomedcentral.com/" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="247,59,296,75" alt="search" title="search" href="http://www.diagnosticpathology.org/search" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="301,59,405,75" alt="submit a manuscript" title="submit a manuscript" href="http://www.diagnosticpathology.org/manuscript" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="415,59,458,75" alt="register" title="register" href="http://www.diagnosticpathology.org/register" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="185,59,244,75" alt="this article" title="this article" href="http://dx.doi.org/10.1186/1746-1596-6-S1-S18" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,480,75" alt="Diagnostic Pathology" title="Diagnostic Pathology" href="http://www.diagnosticpathology.org/" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal Front Page" title="Journal Front Page" href="http://www.diagnosticpathology.org/" ref="reftype=publisher&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-en page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><div><span class="cit">Diagn Pathol. 2011; 6(Suppl 1): S18. </span></div><div><span class="fm-vol-iss-date">Published online Mar 30, 2011. </span>  <span class="doi">doi:&nbsp; <a href="http://dx.doi.org/10.1186%2F1746-1596-6-S1-S18" target="pmc_ext" ref="reftype=other&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1186/1746-1596-6-S1-S18</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3073211</span></div></div></div></div></div><h1 class="content-title">Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Markiewicz%20T%5Bauth%5D">Tomasz Markiewicz</a><sup><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/corrauth.gif" alt="corresponding author"></sup><sup>#</sup><sup>1,</sup><sup>2</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="id838155_ai" style="display:none"><div class="fm-affl"><sup>1</sup>Institute of Theory of Electrical Engineering, Measurement and Information System, Warsaw University of Technology, ul. Koszykowa 75, 00-662 Warsaw, Poland</div><div class="fm-affl"><sup>2</sup>Department of Pathology, Military Institute of Medicine, ul. Szaserow 128, 04-141 Warsaw, Poland</div><div><sup><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div><sup>#</sup>Contributed equally.</div><div><span class="contrib-email" id="A1">Tomasz Markiewicz: <a href="mailto:dev@null" data-email="lp.ude.wp.mei@tweikram" class="oemail">lp.ude.wp.mei@tweikram</a></span> </div></div><div class="togglers"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="pmctoggle" rid="id838155_ai">Author information <span>►</span></a> <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="pmctoggle" rid="id838155_an">Article notes <span>►</span></a> <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="pmctoggle" rid="id838155_cpl">Copyright and License information <span>►</span></a></div><div class="fm-article-notes fm-panel hide half_rhythm" id="id838155_an" style="display:none"><div class="supplement"><h4>Supplement</h4><div class="supplement-title">Proceedings of the 10th European Congress on Telepathology and 4th International Congress on Virtual Microscopy</div><div class="supplement-editor">Klaus Kayser, Arvydas Laurinavicius and Gian Kayser</div><div class="supplement-sponsor">Sponsorship for publication of these proceedings has been provided by the International Academy of Telepathology (IAT), COST-Action IC0604, EU Verein Förderung des Biol.-Techn. Fortschritts in der Medizin, e.V., Heidelberg, DiagnomX GmbH.</div></div><div class="conference"><h4>Conference</h4><div class="conf-name">The 10th European Congress on Telepathology and 4th International Congress on Virtual Microscopy</div><div class="conf-date">1–3 July 2010</div><div class="conf-loc">Vilnius, Lithuania</div></div></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="id838155_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="http://www.ncbi.nlm.nih.gov/pmc/about/copyright.html">Copyright</a>  ©2011 Markiewicz; licensee BioMed Central Ltd.</div><div class="fm-copyright half_rhythm">This is an open access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div class="links-box fm-panel whole_rhythm"><div class="small"><div>This article has been <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="__abstractid844242" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__abstractid844242title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3>Background</h3><p id="__p1" class="p p-first-last">The Matlab software is a one of the most advanced development tool for application in engineering practice. From our point of view the most important is the image processing toolbox, offering many built-in functions, including mathematical morphology, and implementation of a many artificial neural networks as AI. It is very popular platform for creation of the specialized program for image analysis, also in pathology. Based on the latest version of Matlab Builder Java toolbox, it is possible to create the software, serving as a remote system for image analysis in pathology via internet communication. The internet platform can be realized based on Java Servlet Pages with Tomcat server as servlet container.</p></div><div id="__sec2" class="sec"><h3>Methods</h3><p id="__p2" class="p p-first-last">In presented software implementation we propose remote image analysis realized by Matlab algorithms. These algorithms can be compiled to executable <em>jar</em> file with the help of Matlab Builder Java toolbox. The Matlab function must be declared with the set of input data, output structure with numerical results and Matlab web figure. Any function prepared in that manner can be used as a Java function in Java Servlet Pages (JSP). The graphical user interface providing the input data and displaying the results (also in graphical form) must be implemented in JSP. Additionally the data storage to database can be implemented within algorithm written in Matlab with the help of Matlab Database Toolbox directly with the image processing. The complete JSP page can be run by Tomcat server.</p></div><div id="__sec3" class="sec"><h3>Results</h3><p id="__p3" class="p p-first-last">The proposed tool for remote image analysis was tested on the Computerized Analysis of Medical Images (CAMI) software developed by author. The user provides image and case information (diagnosis, staining, image parameter etc.). When analysis is initialized, input data with image are sent to servlet on Tomcat. When analysis is done, client obtains the graphical results as an image with marked recognized cells and also the quantitative output. Additionally, the results are stored in a server database. The internet platform was tested on PC Intel Core2 Duo T9600 2.8GHz 4GB RAM server with 768x576 pixel size, 1.28Mb tiff format images reffering to meningioma tumour (x400, Ki-67/MIB-1). The time consumption was as following: at analysis by CAMI, locally on a server – 3.5 seconds, at remote analysis – 26 seconds, from which 22 seconds were used for data transfer via internet connection. At jpg format image (102 Kb) the consumption time was reduced to 14 seconds.</p></div><div id="__sec4" class="sec sec-last"><h3>Conclusions</h3><p id="__p4" class="p p-first-last">The results have confirmed that designed remote platform can be useful for pathology image analysis. The time consumption is depended mainly on the image size and speed of the internet connections. The presented implementation can be used for many types of analysis at different staining, tissue, morphometry approaches, etc. The significant problem is the implementation of the JSP page in the multithread form, that can be used parallelly by many users. The presented platform for image analysis in pathology can be especially useful for small laboratory without its own image analysis system.</p></div></div></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Background</h2><p id="__p5" class="p p-first">The image analysis in pathology is a very wide and differentiated topic focusing interests of many researchers [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B1" rid="B1" class="bibr popnode tag_hotlink tag_tooltip jig-ncbipopper" id="__tag_216620700" role="button" aria-expanded="false" aria-haspopup="true">1</a>-<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B6" rid="B6" class="bibr popnode tag_hotlink tag_tooltip jig-ncbipopper" id="__tag_319586098" role="button" aria-expanded="false" aria-haspopup="true">6</a>]. The digital microscopic images in pathology are very different, according to the type of tissue, disease, used staining, and required type of analysis (e.g. quantitative, morphometric). Matlab software [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>] is a one of the most developed tool for application in engineering practice and can be useful in this topic. Among many different technical aspects, it supports image processing, giving many built-in functions, especially directed to mathematical morphology and offering implementation of many artificial neural networks, that can be used together with morphological processing for recognition of the cells. Thanks to these features, Matlab is a popular platform for creation of specialized program for image analysis, also in pathology [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B8" rid="B8" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">8</a>].</p><p id="__p6">In the last years the Computerized Analysis of Medical Images (CAMI) software, written in Matlab, was designed and developed by us in the Military Institute of Medicine, Warsaw, Poland [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B9" rid="B9" class="bibr popnode tag_hotlink tag_tooltip jig-ncbipopper" id="__tag_319586099" role="button" aria-expanded="false" aria-haspopup="true">9</a>,<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B10" rid="B10" class="bibr popnode tag_hotlink tag_tooltip jig-ncbipopper" id="__tag_319586100" role="button" aria-expanded="false" aria-haspopup="true">10</a>]. It was used in several research projects as a tool for automatic quantitative and morphometric image analysis. The main directions of application were breast cancer (ER/PR receptors), tumours of the nervous system (neuroblastoma, meningioma, oligodendroglioma – Ki-67 staining) and gastric inflammatory (chromogranin, serotonin and somatostatin antibodies). The positive results of CAMI application inclined us to idea to build internet platform offering the remote desktop to CAMI.</p><p id="__p7" class="p p-last">The latest version of Matlab offers tools for automatic building the executable files of our functions designed in Matlab code by using Java language. With the help of Matlab Builder Java toolbox [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>] it is possible to build executable <em>jar</em> files for use by Java Servlet Pages (JSP) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B11" rid="B11" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">11</a>] in internet www server as a servlet (program executed on server). However example application (existing in internet and presented in the Mathworks web side [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>]) restrict to using Matlab for numerical calculation only or sometimes with graphical visualization, completely omitting the problem of analysis of the images supplied by the remote user. Only one example software (HDRExplorer [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B12" rid="B12" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">12</a>]) with the analysis of the image sequence can be found, but this system works only with the data files stored on the server, not on the user remote station. The image analysis in pathology requires uploading the image to perform the analysis. In this paper the practical realization of image processing in pathology at using Matlab functions by JSP and Tomcat server [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B13" rid="B13" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">13</a>] for is presented.</p></div><div id="__sec6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec6title">Implementation</h2><p id="__p8" class="p p-first">In this study the author proposes to build the remote platform for image analysis based on Matlab function, JSP and Tomcat server. These tools with additional database provide all functions required to realize the task. The basic requirements of the programs are as follows:</p><p id="__p9">• <em>Java</em> Runtime Environment v. 6 or higher</p><p id="__p10">• Tomcat 6.0.20 or higher</p><p id="__p11">• Matlab 7.9 or higher</p><p id="__p12">• Matlab Toolboxes: MATLAB Compiler v. 4.11, MATLAB Builder JA v. 2.0.4, Database Toolbox v. 3.6 and Image Processing Toolbox v. 6.4.or higher.</p><p id="__p13">The main scheme of the platform containing Tomcat server as a container of system’s servlet, the servlet itself written in JSP language, Matlab and data base is presented on the fig. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2em;">fig.1.</span></span><span>1</span></a>. The remote user with the help of web browser connects with the system server via internet. Any web explorer, such as Mozilla Firefox, Internet Explorer etc. can be used. The Tomcat server is a provider of internet service and works as a servlet container. The web page viewed by the remote user is generated by the application servlet written in JSP language. This servlet realizes response and sends any data from web browser, connecting with Matlab Runtime Library [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>] for running image analysis functions. The storage of any data in the data base is controlled by Matlab functions. This data base can be situated in the system server or in a other computer, connected with server e.g. via LAN. The numerical and graphical results of image analysis are returned from Matlab functions to the application servlet and transmitted via internet to web browser. All tools of image analysis platform can be run on the system server and any operation performed on the data can be realized on this computer. The alternative approach with data base placed on the other computer connected with server e.g. via LAN can be also applied.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/1746-1596-6-S1-S18-1.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="/pmc/articles/PMC3073211/bin/1746-1596-6-S1-S18-1.jpg"></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span>The main scheme of remote image analysis system</span></div></div></div><p id="__p15">The proposed remote system requires the set of specific tools and methods for efficient work in practical application. The main task to solve is the realization of the image uploading and returning the results by remote user, working as one of many users of the system. The appropriate mechanisms for realization of these features of the remote platform are implemented by JSP servlet. The first step done by the user is uploading the image for the analysis. The <em>multipart/form-data</em> data form is one of the most commonly used for submitting files via HTTP. This form offers possibility of uploading image file to the remote post frame. The simple code can be written as</p><p id="__p16"><em>&lt;form name=”PlatformName” action=”System.jsp” method=”post” ENCTYPE=”multipart/form-data” &gt;</em>.</p><p id="__p17">The second step is to send the request from client to server. This action requires implementation of acquiring the image file and transmitting it via internet to JSP servlet. The interpretation of the data can be performed by Matlab function. To make such transmission the image file must be read from multipart request via byte stream and used as a parameter for the Matlab function. The sample code of file reading via stream is as bellow</p><p id="__p18">tmpFile = multipartRequest.getFile(“upload”);</p><p id="__p19">FileInputStream fis = new FileInputStream(multipartRequest.getFile(“upload”));</p><p id="__p20">byte dataBytes[] = new byte[fis.available()];</p><p id="__p21">int byteRead = 0;</p><p id="__p22">int wsk=1;</p><p id="__p23">while (wsk&gt;-1) {</p><p id="__p24">byteRead = fis.read(dataBytes);</p><p id="__p25">wsk=byteRead;</p><p id="__p26"><em>}</em>.</p><p id="__p27">The Matlab function can be run by command</p><p id="__p28">Object[] result =</p><p id="__p29">componentObject.function_name(</p><p id="__p30">1, // Always one -- a structure array</p><p id="__p31">dataBytes,</p><p id="__p32">other parameters….);</p><p id="__p33">returning the results of the executing this function in a <em>result</em> object. The <em>function_name</em> is a name of Matlab function and <em>componentObject</em> is the object created by servlet, containing Matlab functions as object methods. The <em>result</em> object may contain any numerical data, such as file name, size and other, and also Matlab web-figure. This web-figure is a virtual type of classical Matlab figure obtained at the stationary use of application and can be placed in HTML code. This web-figure is equipped with zoom, rotate and move buttons, as shown in Fig. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2.</span></span><span>2</span></a>. In the presented solution the remote image analysis is realized by different Matlab algorithms. These algorithms should be compiled to executable <em>jar</em> file with the help of Matlab Compiler and Matlab Builder JA toolbox. The Matlab function must be declared with set of input data and output structure containing the numerical results and Matlab web-figure. Any function prepared in that manner can be used as a Java function in Java Servlet Pages (JSP). The graphical user interface providing the input data and displaying the results (also in graphical form) must be implemented in JSP.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/1746-1596-6-S1-S18-2.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="/pmc/articles/PMC3073211/bin/1746-1596-6-S1-S18-2.jpg"></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span>The example of the Matlab web figure</span></div></div></div><p id="__p35">The JSP servlet realizes the process of uploading the input data (image and description data), transforming the data from html to Matlab function(s) and back to html and providing any commands initiated by the user. The Matlab functions must be compiled into jar file and also added to Tomcat system servlet.</p><p id="__p36">The JSP servlet can initiate the Matlab functions. These functions are interpreted by <em>Matlab</em> Component <em>Runtime</em> (MCR) Library that must be installed in the system’s server. The second way (more recommended) is installing full Matlab program on the server. Indeed any Matlab functions compiled into jar file are executed by the Matlab library. The designed Matlab function(s) can realize image processing by using any <em>Image processing toolbox</em>[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>] tools and connecting with the data base for storage of the image(s) with the help of <em>Database toolbox</em>[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true"><em>7</em></a>]. The data base can be used for storage of the data description and results, and also for creating the unique name/number for the image file/case number. It is possible also using any other useful Matlab tools, such as neural networks as classifiers or statistic analyzers [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7" rid="B7" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">7</a>]. In the presented solution Matlab can offer many tools without necessity to implement in the other language for remote using.</p><p id="__p37">The very important problem in programming of a remote platform for image analysis is parallel computing. The data provided by the user should be visible only by owner and not by the other users. The practical solution of this requirement can be realized by initializing the sessions. Any data and image uploaded by the user, as well as the results of image analysis should be stored only in a specific session initiated only for this user. The setting and getting of the selected attributes can be realized by the sample code</p><p id="__p38">session.setAttribute( " AttribureName ","NewValue")</p><p id="__p39">session.getAttribute("AttribureName").</p><p id="__p40">The same way we use Matlab web-figure. For HTML code generation in the JSP servlet can use command</p><p id="__p41">&lt;wf:web-figure name="myFigure" scope="session" root="WebFigures"&gt;&lt;/wf:web-figure&gt;</p><p id="__p42">where scope parameter controls the process of getting the web-figure from the user session. The scheme of the parallel computation on the remote platform is presented in the fig. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2em;">fig.3</span></span><span>3</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/1746-1596-6-S1-S18-3.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="/pmc/articles/PMC3073211/bin/1746-1596-6-S1-S18-3.jpg"></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span>The scheme of the parallel computing in the remote platform</span></div></div></div><p id="__p44" class="p p-last">The final problem is to provide the synchronization of the connection to data base when it is necessary. This case appears when the user’s session adding the new case to the database, requesting the unique number for it. At any time only one connection channel can be open and still will be closed any operation must be thread-safe by using <em>synchronized</em> command.</p></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Results</h2><p id="__p45" class="p p-first">The proposed tool for remote image analysis was tested by using the Computerized Analysis of Medical Images (CAMI) software developed by author. The user provides image and case information (diagnosis, staining, image parameter etc.) and when analysis is initialized, the input data and image are sent to servlet on Tomcat. There exist two cases of connection with servlet. The first one is provided for image upload and viewed in Matlab web-figure. The second one is for uploading the image (if it wasn’t upload earlier) as well as running the image analysis. When analysis is done, client obtains the results in the form of annotated image with marked recognized cells and quantitative measures. Additionally, the results are stored in server database.</p><p id="__p46">One of the practical question is how to store images in the data base. In CAMI realization the image file is saved in the fixed directory on the server. When image is uploaded only for viewing it, the Matlab function generates 10 digits random number used as a file name and returns it to the user session. When analysis is run, the image processing function gets this image name as well as case description data. The program performs image analysis and when it was successfully done the case will be added to the data base. The function initiates the connection with the data base, adds this new case together with the results of analysis and obtains the new unique etiquette for the processed image. Subsequently, the connection is lost, 10 digits name of the image file is changed into the new unique name and finally web-figure and numerical results are returned to the appropriate user session. After finishing the analysis, the servlet generates HTML code with results. This code is transmitted to the user web browser and the results of image analysis are shown.</p><p id="__p47" class="p p-last">The internet platform was tested on PC Intel Core2 Duo T9600 2.8GHz 4GB RAM server with 768x576 pixel size, 1.28Mb tiff format images with meningioma tumour (x400, Ki-67/MIB-1). The consumption time was as follow: for analysis by CAMI locally on server – 3.5 seconds, for remote analysis – 26 seconds which covers 22 seconds for data transfer via internet connection. For jpg format image (768x576 pixel size, 102 Kb) the consumption time was reduced to 14 seconds. The system can analyze any image formats recognized by Matlab <em>imread</em> function. The user connection speed was 2 Mb/s for download and 0.5 Mb/s for upload.</p></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Discussion</h2><p id="__p48" class="p p-first">The pathological image analysis is a one of the possible specific direction of medical image processing. The considered images represent practically infinite diversity regarding the types of a tissue or likewise types of staining. This differences are the reason of developing many methods and approaches to the computerized image analysis.</p><p id="__p49">Currently we can find many algorithms and languages used for analysis of these images. One of the most advanced tools in the image analysis is Matlab with Image processing toolbox. This program offers many functions realizing very useful operations, forming the advanced mathematical tools. For scientists involved in research of image processing it is a very popular language allowing the implementation and development of many advanced approaches to the image analysis. However, the remote image processing based on Matlab functions via internet is still in early stage of development. One example of image analysis by Matlab with web browser can be found in the paper [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B12" rid="B12" class="bibr popnode jig-ncbipopper" role="button" aria-expanded="false" aria-haspopup="true">12</a>]. Unfortunately, this example works only with the image files stored on server, not with the images provided by user from the remote computer. This solution provides no tools for data storage in data base and parallel computing.</p><p id="__p50">In this paper the approach based on Tomcat server, JSP servlet and connection to data base directly from the Matlab function is proposed. The practical realization of this approach requires finding many keys and solving some problems for offering the practical and useful tool. The presented realization allows uploading the image file from remote computer to servlet and Matlab platform, the connection to data base for data storing and mechanism of parallel computing. To offer the platform for many users together, the session, as a user data container, was applied.</p><p id="__p51" class="p p-last">The other significant feature of the proposed solution is that the processes of image analysis and stored data aren’t visible for remote users. This provides high security of the proposed system and makes the system scalable. The data base can be hidden in the local LAN network and a single process of image analysis can be also moved to the other computers/cores.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Conclusions</h2><p id="__p52" class="p p-first">The results have confirmed that designed remote platform can be useful in pathology image analysis. The platform is easy to be implemented with the help of Matlab and free programs, such as Java and Tomcat. The data base can be also built using any other popular software e.g. Oracle, MS Access, SQL etc. and runs on the ODBC protocol in MS Windows.</p><p id="__p53" class="p p-last">The consumption time depends mainly on the image size and speed of internet connections. The presented implementation can be used for many types of analysis (different staining, tissue, morphometry approaches etc.). The significant question is implementation of the JSP page in the multisession form, that can be used parallel by many users. The presented platform for image analysis in pathology can be especially useful for laboratory having no image analysis system.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Availability and requirements</h2><p id="__p54" class="p p-first">• Project name: Computerized Analysis of Medical Images</p><p id="__p55">• Project home page: <a href="http://www.cami.iem.pw.edu.pl/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.cami.iem.pw.edu.pl</a></p><p id="__p56">• Operating system(s): MS Windows</p><p id="__p57">• Programming language: Matlab, Java, Java Servlet Pages</p><p id="__p58">• Other requirements: Java Runtime Environment 6 or higher, Tomcat 6.0.20 or higher, Matlab 7.9 or higher, Matlab Toolboxes: MATLAB Compiler 4.11, MATLAB Builder JA 2.0.4, Database Toolbox 3.6 and Image Processing Toolbox 6.4.or higher</p><p id="__p59">• License: Matlab license with depicted toolboxes, Java and Tomcat – GNU licence.</p><p id="__p60" class="p p-last">• Any restrictions to use by non-academics: licence needed.</p></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Competing interests</h2><p id="__p61" class="p p-first-last">The authors declare that they have no competing interests.</p></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Authors' contributions</h2><p id="__p62" class="p p-first-last">TM – 100 % contribution of the presented paper.</p></div><div id="__sec13" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Acknowledgements</h2><!--/article/back/sec/--><p id="__p63" class="p p-first">This work was supported by grant of the European Union in the framework of the European Social Fund through the Warsaw University of Technology Development Programme and cooperation with COST Action IC0604.</p><p id="__p64" class="p p-last">This article has been published as part of <em>Diagnostic Pathology</em> Volume 6 Supplement 1, 2011: Proceedings of the 10th European Congress on Telepathology and 4th International Congress on Virtual Microscopy. The full contents of the supplement are available online at <a href="http://www.diagnosticpathology.org/supplements/6/S1" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.diagnosticpathology.org/supplements/6/S1</a></p></div><div id="__ref-listid839903" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__ref-listid839903title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">Bartels PH, Thompson D, Montironi R, Mariuzzi G, Hamilton PW. Automated reasoning system in histopathologic diagnosis and prognosis of prostate cancer and its precursors. <span><span class="ref-journal">Eur Urol. </span>1996;<span class="ref-vol">30</span>:222–233.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/8875204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B2"><span class="mixed-citation">Kayser G, Radziszowski D, Bzdyl P, Werner M, Kayser K.  <span class="ref-journal">Conf. International Society for Cellular Oncology (ISCO)</span> Belfast; 2005. Eamus – internet based platform for automated quantitative measurements in immunohistochemistry.</span></li><li id="B3"><span class="mixed-citation">Kayser G, Radziszowski D, Bzdyl P, Sommer R, Kayser K. Theory and implementation of an electronic, automated measurement system for images obtained from immunohistochemically stained slides. <span><span class="ref-journal">Anal Quant Cytol. </span>2006;<span class="ref-vol">28</span>:27–38.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16566277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B4"><span class="mixed-citation">Kim YJ, Romeike BFM, Uszkoreit J, Feiden W. Automated nuclear segmentation in the determination of the Ki-67 labeling index in meningiomas. <span><span class="ref-journal">Clin Neuropathol. </span>2006;<span class="ref-vol">25</span>(2):67–73.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/16550739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B5"><span class="mixed-citation">Markiewicz T, Osowski S, Patera J, Kozlowski W. Image processing for accurate recognition and counting of cells of the histological slides. <span><span class="ref-journal">Anal Quant Cytol. </span>2006;<span class="ref-vol">28</span>:281–292.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/17067010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B6"><span class="mixed-citation">Markiewicz T, Wisniewski P, Osowski S, Patera J, Kozlowski W, Koktysz R. Comparative analysis of the methods for accurate recognition of cells in the nuclei staining of the Ki-67 in neuroblastoma and ER/PR status staining in breast cancer. <span><span class="ref-journal">Anal Quant Cytol. </span>2009;<span class="ref-vol">31</span>(1):49–62.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19320193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B7"><span class="mixed-citation">MATLAB. <a href="http://www.mathworks.com/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.mathworks.com</a></span></li><li id="B8"><span class="mixed-citation">Demirkaya O, Asyali MH, Sahoo PK.  <span class="ref-journal">Image Processing with MATLAB: Applications in Medicine and Biology.</span> Boca Raton: CRC Press; 2009. </span></li><li id="B9"><span class="mixed-citation">Grala B, Markiewicz T, Kozłowski W, Osowski S, Słodkowska J, Papierz W. New automated image analysis method for assesment of Ki-67 labeling index in meningiomas. <span><span class="ref-journal">Folia Histochem Cyto. </span>2009;<span class="ref-vol">47</span>(4):587–592. doi: 10.2478/v10042-008-0098-0.</span>  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/20430724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span> <span class="nowrap ref crossref">[<a href="http://dx.doi.org/10.2478%2Fv10042-008-0098-0" target="pmc_ext" ref="reftype=other&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">Cross Ref</a>]</span></span></li><li id="B10"><span class="mixed-citation">Markiewicz T, Grala B, Kozlowski W, Osowski S. Computer system for cell counting in selected brain tumors at Ki-67 immunohistochemical staining. <span class="ref-journal">Anal Quant Cytol Histol.</span>   in press .  <span class="nowrap ref pubmed">[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21456344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></li><li id="B11"><span class="mixed-citation">JAVA. <a href="http://java.sun.com/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://java.sun.com</a></span></li><li id="B12"><span class="mixed-citation">Webb P. Rendering High Dynamic Range Images on the Web. <span><span class="ref-journal">MATLAB Digest. </span>2008;<span class="ref-vol">17</span>(4):1–5.</span></span></li><li id="B13"><span class="mixed-citation">Tomcat. <a href="http://tomcat.apache.org/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://tomcat.apache.org</a></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-1"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__abstractid844242title">Abstract</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec5title">Background</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec6title">Implementation</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec7title">Results</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec8title">Discussion</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec9title">Conclusions</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec10title">Availability and requirements</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec11title">Competing interests</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec12title">Authors' contributions</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__sec13title">Acknowledgements</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#__ref-listid839903title">References</a></li></ul></div></div><!--post-content--><hr class="whole_rhythm no_bottom_margin"><div class="courtesy-note no_margin small">Articles from <span class="acknowledgment-journal-title">Diagnostic Pathology</span> are provided here courtesy of <strong>BioMed Central</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="try-button"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/?report=reader"><img src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/3838809" alt="PubReader format: click here to try"></a></div><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/epub/">ePub (beta)</a></li> | <li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/pdf/1746-1596-6-S1-S18.pdf">PDF (379K)</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related citations in PubMed</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter" toggles="ui-portlet_content-2" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-2" aria-hidden="false"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/17280615" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">NeuroTerrain--a client-server system for browsing 3D biomedical image data sets.</a><span class="source">[BMC Bioinformatics. 2007]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Gustafson C, Bug WJ, Nissanov J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Bioinformatics. 2007 Feb 5; 8:40. Epub 2007 Feb 5.</em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/15000350" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The virtual microscope.</a><span class="source">[IEEE Trans Inf Technol Biomed. 2003]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Catalyürek U, Beynon MD, Chang C, Kurc T, Sussman A, Saltz J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Inf Technol Biomed. 2003 Dec; 7(4):230-48. </em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/14512356" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The design of Jemboss: a graphical user interface to EMBOSS.</a><span class="source">[Bioinformatics. 2003]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Carver T, Bleasby A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Bioinformatics. 2003 Sep 22; 19(14):1837-43. </em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/15847374" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">From telepathology to virtual pathology institution: the new world of digital pathology.</a><span class="source">[Rom J Morphol Embryol. 1999]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Kayser K, Kayser G, Radziszowski D, Oehmann A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Rom J Morphol Embryol. 1999-2004; 45:3-9. </em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/19622123" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Digital image management project for dermatological health care environments: a new dedicated software and review of the literature.</a><span class="source">[Skin Res Technol. 2009]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Rubegni P, Nami N, Poggiali S, Tataranno D, Fimiani M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Skin Res Technol. 2009 May; 15(2):148-56. </em></div></li></ul><a class="seemore" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&cmd=link&linkname=pubmed_pubmed_reviews&uid=21489188&log$=relatedreviews&logdbfrom=pmc" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&cmd=link&linkname=pubmed_pubmed&uid=21489188&log$=relatedarticles&logdbfrom=pmc" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a></div></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" id="Shutter" toggles="ui-portlet_content-3" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-3" aria-hidden="false"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3656801/" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Validation of various adaptive threshold methods of segmentation applied to follicular lymphoma digital images stained with 3,3'-Diaminobenzidine&amp;Haematoxylin</a><span class="source">[Diagnostic Pathology. ]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Korzynska A, Roszkowiak L, Lopez C, Bosch R, Witkowski L, Lejeune M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Diagnostic Pathology. 848</em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3584735/" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Validation of a new classifier for the automated analysis of the human epidermal growth factor receptor 2 (HER2) gene amplification in breast cancer specimens</a><span class="source">[Diagnostic Pathology. ]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Furrer D, Jacob S, Caron C, Sanschagrin F, Provencher L, Diorio C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Diagnostic Pathology. 817</em></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073214/" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Accuracy of a remote quantitative image analysis in the whole slide images</a><span class="source">[Diagnostic Pathology. ]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Słodkowska J, Markiewicz T, Grala B, Kozłowski W, Papierz W, Pleskacz K, Murawski P. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Diagnostic Pathology. 6(Suppl 1)S20</em></div></li></ul><a class="seemore" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/citedby/" ref="reftype=relart&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a></div></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Links</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" id="Shutter" toggles="ui-portlet_content-4" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pmc/?Db=medgen&DbFrom=pmc&Cmd=Link&LinkName=pmc_medgen&IdsFromResult=3073211" ref="reftype=MedGen&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div></div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="http://www.ncbi.nlm.nih.gov/pubmed/21489188/" ref="reftype=PubMed&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div></div></li></ul></div></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter" toggles="ui-portlet_content-5" aria-disabled="false" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a></div><div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line" role="button" aria-expanded="false" aria-haspopup="true"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="http://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=1408583952690305">Using MATLAB software with Tomcat server and Java platform for remote image anal...</a><div class="ralinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-ralinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology<div class="brieflinkpopdesc">Diagnostic Pathology. 2011; 6(Suppl 1)S18</div></div><div class="tertiary">PMC</div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="http://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a></div></div></div></div>
        
        <div id="tags_container" data-domain="diagpath" data-accid="PMC3073211" data-aiid="3073211" data-md5="676ceddc8e4c32acea8f394c80ca83db"><!--?xml version="1.0" encoding="UTF-8"?-->
<div><div class="portlet brieflink pmc_para_cit" id="crb--__p5" name="crb--__p5" rid="__p5" style="position: absolute; visibility: visible; width: 370px; top: 1380px;"><div class="portlet_head"></div><div class="portlet_content"><ul><li class="two_line" reference_id="B1"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8875204/" ref="reftype=pubmed&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Automated reasoning system in histopathologic diagnosis and prognosis of prostate cancer and its precursors.</a><span class="one_line_source">[Eur Urol.  1996]</span><div class="alt-note"><div class="authors">Bartels PH, Thompson D, Montironi R, Mariuzzi G, Hamilton PW</div><div class="citation">Eur Urol. 1996; 30(2):222-33.</div></div></li><li class="two_line" reference_id="B6"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19320193/" ref="reftype=pubmed&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comparative analysis of methods for accurate recognition of cells through nuclei staining of Ki-67 in neuroblastoma and estrogen/progesterone status staining in breast cancer.</a><span class="one_line_source">[Anal Quant Cytol Histol.  2009]</span><div class="alt-note"><div class="authors">Markiewicz T, Wisniewski P, Osowski S, Patera J, Kozlowski W, Koktysz R</div><div class="citation">Anal Quant Cytol Histol. 2009 Feb; 31(1):49-62.</div></div></li></ul></div></div><div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 370px; top: 1589px;"><div class="portlet_head"></div><div class="portlet_content"><ul><li class="two_line" reference_id="B9"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20430724/" ref="reftype=pubmed&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">New automated image analysis method for the assessment of Ki-67 labeling index in meningiomas.</a><span class="one_line_source">[Folia Histochem Cytobiol.  2009]</span><div class="alt-note"><div class="authors">Grala B, Markiewicz T, Kozłowski W, Osowski S, Słodkowska J, Papierz W</div><div class="citation">Folia Histochem Cytobiol. 2009; 47(4):587-92.</div></div></li><li class="two_line" reference_id="B10"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21456344/" ref="reftype=pubmed&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Computer system for cell counting in selected brain tumors at Ki-67 immunohistochemical staining.</a><span class="one_line_source">[Anal Quant Cytol Histol.  2010]</span><div class="alt-note"><div class="authors">Markiewicz T, Grala B, Kozlowski W, Osowski S</div><div class="citation">Anal Quant Cytol Histol. 2010 Dec; 32(6):323-32.</div></div></li></ul></div></div></div>
</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-B1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Automated reasoning system in histopathologic diagnosis and prognosis of prostate cancer and its precursors.<div class="authors">Bartels PH, Thompson D, Montironi R, Mariuzzi G, Hamilton PW</div><div class="citation">Eur Urol. 1996; 30(2):222-33.</div><p>[<a href="http://www.ncbi.nlm.nih.gov/pubmed/8875204/">PubMed</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B1">Ref list</a>]</p></div><div id="body-link-popper-B6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Comparative analysis of methods for accurate recognition of cells through nuclei staining of Ki-67 in neuroblastoma and estrogen/progesterone status staining in breast cancer.<div class="authors">Markiewicz T, Wisniewski P, Osowski S, Patera J, Kozlowski W, Koktysz R</div><div class="citation">Anal Quant Cytol Histol. 2009 Feb; 31(1):49-62.</div><p>[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19320193/">PubMed</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B6">Ref list</a>]</p></div><div id="body-link-popper-B7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="mixed-citation">MATLAB. <a href="http://www.mathworks.com/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://www.mathworks.com</a></span> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B7">Ref list</a>]</div><div id="body-link-popper-B8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="mixed-citation">Demirkaya O, Asyali MH, Sahoo PK.  <span class="ref-journal">Image Processing with MATLAB: Applications in Medicine and Biology.</span> Boca Raton: CRC Press; 2009. </span> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B8">Ref list</a>]</div><div id="body-link-popper-B9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">New automated image analysis method for the assessment of Ki-67 labeling index in meningiomas.<div class="authors">Grala B, Markiewicz T, Kozłowski W, Osowski S, Słodkowska J, Papierz W</div><div class="citation">Folia Histochem Cytobiol. 2009; 47(4):587-92.</div><p>[<a href="http://www.ncbi.nlm.nih.gov/pubmed/20430724/">PubMed</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B9">Ref list</a>]</p></div><div id="body-link-popper-B10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Computer system for cell counting in selected brain tumors at Ki-67 immunohistochemical staining.<div class="authors">Markiewicz T, Grala B, Kozlowski W, Osowski S</div><div class="citation">Anal Quant Cytol Histol. 2010 Dec; 32(6):323-32.</div><p>[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21456344/">PubMed</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B10">Ref list</a>]</p></div><div id="body-link-popper-B11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="mixed-citation">JAVA. <a href="http://java.sun.com/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://java.sun.com</a></span> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B11">Ref list</a>]</div><div id="body-link-popper-B12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="mixed-citation">Webb P. Rendering High Dynamic Range Images on the Web. <span><span class="ref-journal">MATLAB Digest. </span>2008;<span class="ref-vol">17</span>(4):1–5.</span></span> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B12">Ref list</a>]</div><div id="body-link-popper-B13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="mixed-citation">Tomcat. <a href="http://tomcat.apache.org/" ref="reftype=extlink&amp;article-id=3073211&amp;issue-id=195730&amp;journal-id=414&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://tomcat.apache.org</a></span> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#B13">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="http://www.ncbi.nlm.nih.gov/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/pmc/">PubMed Central (PMC)</a></span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="http://www.ncbi.nlm.nih.gov/sites/ehelp?&Ncbi_App=pmc&Db=pmc&Page=literature&Snapshot=/projects/PMC/PMCViewer@4.28&Time=2014-08-20T22:08:15-04:00&Host=ptpmc101">Write to the Help Desk</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"><!--?xml version="1.0" encoding="UTF-8"?-->
<div><h2 class="offscreen_noflow">Simple NCBI Directory</h2><ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            
            <li><a href="http://www.ncbi.nlm.nih.gov/education/">NCBI Education</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul><ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            
        <li><a href="http://www.ncbi.nlm.nih.gov/guide/chemicals-bioassays">Chemicals &amp; Bioassays</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/data-software">Data &amp; Software</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/dna-rna">DNA &amp; RNA</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/domains-structures">Domains &amp; Structures</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/genes-expression">Genes &amp; Expression</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/genetics-medicine">Genetics &amp; Medicine</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/genomes-maps">Genomes &amp; Maps</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/homology">Homology</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/literature">Literature</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/proteins">Proteins</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/sequence-analysis">Sequence Analysis</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/taxonomy">Taxonomy</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/training-tutorials">Training &amp; Tutorials</a></li><li><a href="http://www.ncbi.nlm.nih.gov/guide/variation">Variation</a></li></ul><ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/">PubMed</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/books/">Bookshelf</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/pmc/">PubMed Central</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="http://www.ncbi.nlm.nih.gov/nucleotide/">Nucleotide</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/genome/">Genome</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/snp/">SNP</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/gene/">Gene</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul><ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/gtr/">Genetic Testing Registry</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/refseq/">Reference Sequences</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/mapview/">Map Viewer</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/genome/guide/human/">Human Genome</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/Traces/sra/">Sequence Read Archive</a></li>
        </ul><ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/About/">About NCBI</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/research/">Research at NCBI</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/news/">NCBI News</a></li>
            <li><a href="http://www.ncbi.nlm.nih.gov/Ftp/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm" role="button" aria-expanded="false" aria-haspopup="true" target="_blank">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi" role="button" aria-expanded="false" aria-haspopup="true" target="_blank">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm" role="button" aria-expanded="false" aria-haspopup="true" target="_blank">NCBI on YouTube</a></li>
        </ul></div>
</div><script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Copyright</a> | <a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="http://www.ncbi.nlm.nih.gov/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="http://www.ncbi.nlm.nih.gov/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/InstrumentPageStarterJS.js"> </script>    
    <script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow" aria-live="assertive"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant=""></ul><div class="ui-ncbiautocomplete-actions shadow"><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="ui-ncbiautocomplete-link-pref">Preferences</a><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="ui-ncbiautocomplete-link-off">Turn off</a></div></div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div><div style="display: none;">External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=diagpath&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC3073211/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FB3FFC3CFF1E01_0240SID /projects/PMC/PMCViewer@4.28 ptpmc101 v4.1.r435145 Wed, May 14 2014 02:50:46 -->

<script type="text/javascript" src="./Using MATLAB software with Tomcat server and Java platform for remote image analysis in pathology_files/3911094.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all" tabindex="-1" role="dialog" aria-labelledby="ui-dialog-title-epubDialog" style="display: none; z-index: 1000; outline: 0px;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span class="ui-dialog-title" id="ui-dialog-title-epubDialog">Making articles easier to read in PMC</span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073211/#" class="ui-dialog-titlebar-close ui-corner-all" role="button"><span class="ui-icon ui-icon-closethick">close</span></a></div><div id="epubDialog" style="" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul></body></html>